tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_files_groupownership/IncludeConfig_is_other.fail.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_pci-dss
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol
+
+# Check rsyslog.conf with root group-owner log from rules and
+# non root group-owner log from $IncludeConfig fails.
+
+source ../rsyslog_log_utils.sh
+
+GROUP_TEST=testssg
+groupadd $GROUP_TEST
+
+GROUP_ROOT=root
+
+# setup test data
+create_rsyslog_test_logs 2
+
+# setup test log files ownership
+chgrp $GROUP_ROOT ${RSYSLOG_TEST_LOGS[0]}
+
```

Comment:
I've added these empty lines for test scenario readability, while I don't mind removing them if needed.

Reply:
I get it and most of empty lines improves readability, but e.g. these between `chgrp` I'd say divide code that should be together, because they belong to same part of test.

Reply:
+ with a lot of empty lines I have feeling that readability is getting worse, so it makes sense to divide some parts of code, but not too much

Reply:
@mildas thanks for the explanation, I've removed empty lines including those after shebang.